tests/test_structure/test_create_structure.py

```diff
@@ -0,0 +1,40 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.structure.create_structure_fixtures import (
+    office_structure_mutation_query,
+    office_structure_mutation_response,
+    empty_field_structure_mutation
+)
+
+
+class TestCreateOfficeStructure(BaseTestCase):
+
+    def test_structure_creation(self):
+        """
+        Testing for OfficeStructure creation
```

Comment:
Use more descriptive docstring to describe what we are testing for in each test. Go through the codebase to see how the descriptions are written.

Reply:
Done